lib/puppet_litmus/serverspec.rb

```diff
@@ -119,13 +119,19 @@ def bolt_upload_file(source, destination, options = {})
 
     result = upload_file(source, destination, target_node_name, options: options, config: nil, inventory: inventory_hash)
 
-    raise "upload file failed\n======\n#{result}" if result.first['status'] != 'success'
+    result_obj = {
+      exit_code: 0,
+      stdout: (result.first['result']['_output'] unless result.first['result']['_output'].nil?),
```

Comment:
Wasn't sure if this was idiomatic or not, but it was short and didn't require another line of code